README.md

```diff
@@ -1,26 +1,255 @@
+
+To install Raspbian, we recommend following the "Getting started with Raspberry Pi" tutorial below. 
+
+**NOTE: These instructions were tested and verified using Raspbian Stretch. Newer versions of Raspbian (such as Buster) may cause complications with these instructions. We therefore recommend installing Raspbian Stretch instead of the latest/default version of Raspbian, which the tutorial below will recommend. Do not flash NOOBS onto your card; instead download the Raspbian Streth image below and skip the NOOBS step in the tutorial.** 
+
+In addition, we have created a pre-built image with both Raspbian Stretch and ROS already installed. You can download this image as a last resort if you can’t get Raspbian and ROS installed and working on your own, but please note that you will miss out on some the learning opportunities that come with debugging a software installation by doing so. 
+
+- [Tutorial](https://projects.raspberrypi.org/en/projects/raspberry-pi-getting-started) for installing the latest version of Raspbian on a new Raspberry Pi
+- Direct downloadable Raspbian Stretch [image](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip)
+- Custom JPL [image](https://drive.google.com/drive/folders/1RbYWDRthpcktqkPEHb7qVqDiy27EiAc1) with both Raspbian Stretch and ROS pre-built
+
+**NOTE: If you install the Custom JPL image with Raspbian Stretch and ROS pre-built, skip to step 2.4.**
+
+Once you finish the above tutorial and install Raspbian Stretch, you should be able to boot your Raspberry Pi and see a desktop!
+
+
+
+
+
+### 2.2 Installing ROS
+
+
+#### 2.2.1 ROS Melodic on Ubuntu 18.04
+
+Use these instructions if you followed subsection 2.1.1 above.
+
+The below steps are based off of [these instructions](http://wiki.ros.org/melodic/Installation/Ubuntu). Consult them for more details.
+
+Setup
+```
+# Setup your computer to accept software from packages.ros.org
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+
+# set up keys
+sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+```
+
+Installation
+```
+# make sure your Debian package index is up-to-date
+sudo apt update
+
+# desktop-Full Install: (Recommended) : ROS, rqt, rviz, robot-generic libraries, 2D/3D simulators and 2D/3D perception
+sudo apt install ros-melodic-desktop-full
+```
+
+Dependencies for building packages
+```
+# install 'em
+sudo apt install python-rosdep python-rosinstall python-rosinstall-generator python-wstool build-essential
+
+# initialize rosdep
+sudo apt install python-rosdep
+sudo rosdep init
+rosdep update
+
+# sensor msgs dependencies
+rosinstall_generator sensor_msgs --rosdistro melodic --deps --wet-only --tar >melodic-sensor_msgs-wet.rosinstall
+wstool init src melodic-sensor_msgs-wet.rosinstall
+```
+
+#### 2.2.2 ROS Kinetic on Raspbian
```

Comment:
If we make Ubuntu18 the default, then I'm fine saying 'you can do raspbian but we don't have instructions for it so do it at your own risk'. Otherwise we have to explicitly support multiple platforms and then support and maintenance blows up. @ericjunkins thoughts?

Reply:
I agree that a single platform is a good idea. Are we ready to jump to Ubuntu 18.04 though?

I don't have a vote here. I'm way too new to the project, and don't actually have _that_ much xp with raspberry pi's.

Reply:
I'd say so. We gotta stay up-to-date and multiple builders are on 18.04 already. Code is ROS1-version agnostic so far

Reply:
Made 18.04 the default now.